modules/ocf_stats/manifests/prometheus.pp

```diff
@@ -0,0 +1,35 @@
+# prometheus daemon config
+class ocf_stats::prometheus {
+  class { '::prometheus':
+    version        => '2.0.0',
+    alerts         => {},
+    scrape_configs => [
+      { 'job_name' => 'prometheus',
```

Comment:
nit: formatting consistency

Reply:
What specifically do you take issue with? Other than `'labels'` being too far to the left I don't see what's wrong here

Reply:
labels, there's no space after `"alias" => "Prometheus"` that sort of minor thing